packages/synapse-react-client/src/components/PortalFeatureHighlights/PortalFeatureHighlights.tsx

```diff
@@ -7,79 +7,86 @@ export type PortalFeatureHighlightsProps = {
   title?: string
   image?: string
   buttonText?: string
-  summaryText?: string | React.ReactNode
+  summaryText?: React.ReactNode
   link?: string
 }
 
 const PortalFeatureHighlights = (props: PortalFeatureHighlightsProps) => {
   const { reverseOrder, image, title, buttonText, summaryText, link } = props
+
   return (
-    <Grid
-      container
-      gap={{ xs: '38px', md: '80px' }}
-      padding={{ md: '0 56px 0px 56px' }}
-      justifyContent="center"
-      alignItems={{ md: 'center', lg: 'flex-start' }}
+    <Box
+      sx={{
+        display: 'grid',
+        padding: { xs: '40px', lg: '80px' },
+        gap: { xs: '38px', md: '80px' },
+        gridTemplateColumns: {
+          xs: 'minmax(100px, 1fr)',
+          lg: reverseOrder
+            ? 'minmax(150px, 1fr) minmax(300px, 2fr)'
+            : 'minmax(300px, 2fr) minmax(150px, 1fr)',
+        },
+        gridTemplateAreas: {
+          xs: `'image' 'content'`,
+          lg: reverseOrder ? `'content image'` : `'image content'`,
+        },
```

Comment:
nice use of gridTemplateAreas, it's very clear how the layout is switching between sizes and `reverseOrder` state